.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
 before_install:
   - npm install -g npm@latest
+install:
+  - npm install --global electron
```

Comment:
The issue is that Electron patches the version of node, and even monkey patches some of its standard libraries (most notably `fs`, which is heavily used here), therefore I think it is beneficial to run tests on Electron (we shouldn't drop that).

My proposal for including `electron` as a development dependency is to be able to set it to the current Electron version Etcher is targeting (which is 1.4.4 at the time of this writing). You can of course do something like `npm install -g electron@1.4.4` however we then have to write instructions asking people to install electron globally before being able to run the tests.